routes/index.js

```diff
@@ -178,14 +179,7 @@ module.exports = () => {
 						_profile.bio = account.bio;
 
 						Account.findOne({ username: requestingUser }).then(acc => {
-							if (TRIALMODS.includes(requestingUser)) {
-								try {
-									_profile.lastConnectedIP = '-' + obfIP(_profile.lastConnectedIP);

```

Comment:
lines 181 to 187 are important. They hide the actual IPs of users from Trail Mods. You should not remove these lines.

Reply:
those lines are duplicates of the ones below for the rest of AEM staff. Tried to reduce duplicate code by adding a way for trial mods to have access in the same spot and remove the hard coded role portion.

Reply:
i thought trial mods get to see what AEM sees? according to wiki..?

Reply:
regarding obf ips in profile access

Reply:
It looks like it is redundant. I didn't see this lower portion. This is fine.